src/main.rs

```diff
@@ -15,23 +16,28 @@ const SLEEP_TIME_MS: u64 = 50;
 const MINIDSP_GAIN_MIN: f64 = -127.0;
 const MINIDSP_GAIN_MAX: f64 = 0.0;
 
+// Lazy static needed since Regex::new() is a function call
+lazy_static! {
```

Comment:
Not sure why we make it lazy. It'll be called, and will be called pretty fast. Usually lazy is used for expensive operations that might not be used? 👍

Reply:
Normal "static" doesn't work since Rust won't accept it. Probably due to the size needs to be known at compile time.

I agree that lazy is a bit weird here, any suggestions that still avoids initializing the regex on each call?

Reply:
Uhm how about a `const`? Shouldn't that be possible?

Reply:
const is just search and replace in Rust, the compiler won't accept it... this regex is tearing me apart.